URBNShareKit.podspec

```diff
@@ -10,8 +10,8 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '7.0'
   s.requires_arc = true
 
-  s.source_files = 'Pod/Classes'
+  s.source_files = 'Pod/Classes', 'Pod/Assets'
```

Comment:
Do you need to include the assets directory as a `source_file`? If so, just use a wildcard to get all sub directories in Pod/